swift/IceStorm/weather/Sources/Station/main.swift

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) ZeroC, Inc.
+
+import Ice
+import IceStorm
+
+// CtrlCHandler is a helper class that handles Ctrl+C and similar signals. It must be constructed at the beginning
+// of the program, before creating an Ice communicator or starting any thread.
+let ctrlCHandler = CtrlCHandler()
+
+// Create an Ice communicator. We'll use this communicator to create an object adapter.
```

Comment:
Here the communicator is also used for the proxies, and outgoing connections to IceStorm.